src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -5419,11 +5799,18 @@ public override IEnumerable<IOperation> Children
         {
             get
             {
-                yield return Condition;
-                yield return Body;
-                if (IgnoredCondition != null)
+                IOperation current;
+                if ((current = Condition) != null)
```

Comment:
>if ((current = Condition) != null) [](start = 16, length = 34)

It feels like the order of evaluation between ```Condition``` and the ```Body``` should be dependent on DoLoopKind (i.e. top .vs bottom).

Reply:
@AlekseyTs 2 can't exist at the same time. so output won't be affected I believe? in error case where both condition and ignore condition exist both, I believe evaluation condition first is right?

Reply:
> 2 can't exist at the same time. so output won't be affected I believe? 

Are you saying the ```Condition``` and the ```Body``` are never present at the same time?

Reply:
@AlekseyTs I meant Condition and IgnoredCondition exist at the same time unless it is error case.

Reply:
>  I meant Condition and IgnoredCondition exist at the same time unless it is error case.

I didn't say anything about ```IgnoredCondition```.

Reply:
@AlekseyTs  ah, got confused by what you meant. I understand now.